schematics/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { virtualFs, workspaces } from '@angular-devkit/core';
+import { Tree } from '@angular-devkit/schematics';
+
+// this code is orginal from @schematics/angular
+// we also could include the lib and provide the func getWorkspace from their
```

Comment:
I would definitely recommend to use it directly from `@schematics/angular` 💯

Reply:
Let's discuss this in our session 😀